sdf/CMakeLists.txt

```diff
@@ -20,12 +20,13 @@ endif()
 # descriptions in a map of strings. The parser.cc file uses EmbeddedSdf.hh.
 execute_process(
   COMMAND ${Python3_EXECUTABLE} ${CMAKE_SOURCE_DIR}/sdf/embedSdf.py
+      --output-file "${PROJECT_BINARY_DIR}/src/EmbeddedSdf.cc"
   WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/sdf"
   OUTPUT_FILE "${PROJECT_BINARY_DIR}/src/EmbeddedSdf.cc"
```

Comment:
I think this `OUTPUT_FILE` line needs to be removed, since we are passing `--output-file`. I think this is causing the windows build failure

Reply:
I think you removed the wrong line in your force-push to https://github.com/gazebosim/sdformat/pull/1297/commits/5fbe349793af470b359972c2d0e4e9be5435224d (comparison https://github.com/gazebosim/sdformat/compare/25d793e8149cffb8d5c05b526a31fa030943e952..5fbe349793af470b359972c2d0e4e9be5435224d)

I think the `--output-file "${PROJECT_BINARY_DIR}/src/EmbeddedSdf.cc"` line should remain but the `OUTPUT_FILE "${PROJECT_BINARY_DIR}/src/EmbeddedSdf.cc"` line should be removed

Reply:
Yes, that line looks good now!